ceph/rados/core_workflows.py

```diff
@@ -3148,16 +3151,16 @@ def restart_daemon_services(self, daemon: str, timeout: int = 300):
                         assert entry["status_desc"] != "stopped"
                         log.info(f"{entry['daemon_name']} has started")
                         success = True
-                    except AssertionError:
+                    except Exception:
                         log.info(
                             f"{daemon} daemon {entry['daemon_name']} is yet to restart. "
-                            f"Sleeping for 30 secs"
+                            f"Sleeping for 120 secs"
                         )
                         self.client.exec_command(
                             cmd=f"ceph orch daemon restart {entry['daemon_name']}",
                             sudo=True,
                         )
-                        time.sleep(30)
+                        time.sleep(120)
```

Comment:
let's keep this sleep to 30 secs. this is check interval. let's keep checking often